app/validators/client_details/answers_validator.rb

```diff
@@ -58,7 +58,14 @@ def has_nino_complete?
       return false if applicant.has_nino.blank? && applicant.has_arc.blank?
       return true if applicant.has_nino == 'no'
```

Comment:
Was there a reason the Provider did not state 'no' to the has_nino question? This would have enabled them to proceed I'd thought?